src/Template.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Elastica;
+
+use Elastic\Elasticsearch\Exception\ClientResponseException;
+use Elastic\Elasticsearch\Exception\MissingParameterException;
+use Elastic\Elasticsearch\Exception\ServerResponseException;
+use Elastic\Transport\Exception\NoNodeAvailableException;
+use Elastica\Exception\ClientException;
+use Elastica\Exception\InvalidException;
+
+/**
+ * Elastica legacy index template object.
+ *
+ * @author Dmitry Balabka <dmitry.balabka@gmail.com>
+ *
+ * @deprecated _template have been deprecated in Elasticsearch 7.8 and will be removed in a future version. You should use the IndexTemplate class instead.
```

Comment:
Nit: I think the deprecation happened later that is just when the other templates were introduced but could not find quickly when. So lets keep it.

Reply:
[Depracted since 7.8](https://www.elastic.co/guide/en/elasticsearch/reference/7.8/release-highlights.html)

> Composable index templates replace the previous version of index templates, which are now deprecated. If an index matches both a composable template and a legacy index template, Elasticsearch uses the composable template.

Reply:
Oh, I was not aware. Thanks for the link.